Examples use incorrect package references for C#, Go, and Java

## Description

 The examples in this repository reference incorrect package names for C#, Go, and Java. Users following these examples will encounter build
 failures because the package references don't match the actual published packages.

 ## Summary of Issues

 | Language | Example Reference | Actual Published Package | Status |
 |----------|------------------|-------------------------|--------|
 | **C#** | `Pulumi.Webflow` v1.0.0-alpha.0 | `Community.Pulumi.Webflow` v0.9.0 | ❌ Wrong package name |
 | **Go** | `github.com/jdetmar/...` | `github.com/JDetmar/...` | ❌ Case mismatch |
 | **Java** | `com.pulumi:webflow` | `io.github.jdetmar:pulumi-webflow` | ❌ Wrong groupId/artifactId |
 | Python | `pulumi-webflow` | `pulumi-webflow` | ✅ Correct |

 ## Details

 ### C# (.NET)

 **Current** (`examples/robotstxt/csharp/Webflow.RobotsTxt.csproj`):
 ```xml
 <PackageReference Include="Pulumi.Webflow" Version="1.0.0-alpha.0" />

 Should be:
 <PackageReference Include="Community.Pulumi.Webflow" Version="0.9.0" />

 Error when building:
 error NU1101: Unable to find package Pulumi.Webflow. No packages exist with this id in source(s): nuget.org

 Go

 Current (examples/quickstart/go/go.mod):
 require github.com/jdetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1

 Should be (note uppercase D):
 require github.com/JDetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1

 Go module paths are case-sensitive, so the lowercase jdetmar won't resolve correctly.

 Java

 Current (examples/collection/java/pom.xml):
 <groupId>com.pulumi</groupId>
 <artifactId>webflow</artifactId>
 <version>1.0.0-alpha.0</version>

 Should be:
 <groupId>io.github.jdetmar</groupId>
 <artifactId>pulumi-webflow</artifactId>
 <version><!-- current published version --></version>

 Affected Files

 All C#, Go, and Java example directories likely have this issue:
 - examples/*/csharp/*.csproj
 - examples/*/go/go.mod
 - examples/*/java/pom.xml

 Suggested Fix

 Update the example files to reference the correct published package names for each language.

--- a/examples/asset/csharp/Webflow.Asset.csproj
+++ b/examples/asset/csharp/Webflow.Asset.csproj
@@ -8,6 +8,6 @@
 
   <ItemGroup>
     <PackageReference Include="Pulumi" Version="3.0.0" />
-    <PackageReference Include="Pulumi.Webflow" Version="1.0.0-alpha.0" />
+    <PackageReference Include="Community.Pulumi.Webflow" Version="0.9.0" />
   </ItemGroup>
 </Project>

--- a/examples/asset/go/go.mod
+++ b/examples/asset/go/go.mod
@@ -1,11 +1,11 @@
-module github.com/jdetmar/pulumi-webflow/examples/asset
+module github.com/JDetmar/pulumi-webflow/examples/asset
 
 go 1.24.7
 
 toolchain go1.24.11
 
 require (
-	github.com/jdetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
+	github.com/JDetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
 	github.com/pulumi/pulumi/sdk/v3 v3.212.0
 )
 

--- a/examples/asset/java/pom.xml
+++ b/examples/asset/java/pom.xml
@@ -28,8 +28,8 @@
             <version>${pulumi.version}</version>
         </dependency>
         <dependency>
-            <groupId>com.pulumi</groupId>
-            <artifactId>webflow</artifactId>
+            <groupId>io.github.jdetmar</groupId>
+            <artifactId>pulumi-webflow</artifactId>
             <version>${webflow.version}</version>
         </dependency>
     </dependencies>

--- a/examples/collection/csharp/Webflow.Collection.csproj
+++ b/examples/collection/csharp/Webflow.Collection.csproj
@@ -8,6 +8,6 @@
 
   <ItemGroup>
     <PackageReference Include="Pulumi" Version="3.0.0" />
-    <PackageReference Include="Pulumi.Webflow" Version="1.0.0-alpha.0" />
+    <PackageReference Include="Community.Pulumi.Webflow" Version="0.9.0" />
   </ItemGroup>
 </Project>

--- a/examples/collection/go/go.mod
+++ b/examples/collection/go/go.mod
@@ -1,11 +1,11 @@
-module github.com/jdetmar/pulumi-webflow/examples/collection
+module github.com/JDetmar/pulumi-webflow/examples/collection
 
 go 1.24.7
 
 toolchain go1.24.11
 
 require (
-	github.com/jdetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
+	github.com/JDetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
 	github.com/pulumi/pulumi/sdk/v3 v3.212.0
 )
 

--- a/examples/collection/java/pom.xml
+++ b/examples/collection/java/pom.xml
@@ -28,8 +28,8 @@
             <version>${pulumi.version}</version>
         </dependency>
         <dependency>
-            <groupId>com.pulumi</groupId>
-            <artifactId>webflow</artifactId>
+            <groupId>io.github.jdetmar</groupId>
+            <artifactId>pulumi-webflow</artifactId>
             <version>${webflow.version}</version>
         </dependency>
     </dependencies>

--- a/examples/collectionitem/csharp/Webflow.CollectionItem.csproj
+++ b/examples/collectionitem/csharp/Webflow.CollectionItem.csproj
@@ -8,6 +8,6 @@
 
   <ItemGroup>
     <PackageReference Include="Pulumi" Version="3.0.0" />
-    <PackageReference Include="Pulumi.Webflow" Version="1.0.0-alpha.0" />
+    <PackageReference Include="Community.Pulumi.Webflow" Version="0.9.0" />
   </ItemGroup>
 </Project>

--- a/examples/collectionitem/go/go.mod
+++ b/examples/collectionitem/go/go.mod
@@ -1,11 +1,11 @@
-module github.com/jdetmar/pulumi-webflow/examples/collectionitem
+module github.com/JDetmar/pulumi-webflow/examples/collectionitem
 
 go 1.24.7
 
 toolchain go1.24.11
 
 require (
-	github.com/jdetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
+	github.com/JDetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
 	github.com/pulumi/pulumi/sdk/v3 v3.212.0
 )
 

--- a/examples/collectionitem/java/pom.xml
+++ b/examples/collectionitem/java/pom.xml
@@ -28,8 +28,8 @@
             <version>${pulumi.version}</version>
         </dependency>
         <dependency>
-            <groupId>com.pulumi</groupId>
-            <artifactId>webflow</artifactId>
+            <groupId>io.github.jdetmar</groupId>
+            <artifactId>pulumi-webflow</artifactId>
             <version>${webflow.version}</version>
         </dependency>
     </dependencies>

--- a/examples/page/go/go.mod
+++ b/examples/page/go/go.mod
@@ -1,11 +1,11 @@
-module github.com/jdetmar/pulumi-webflow/examples/page
+module github.com/JDetmar/pulumi-webflow/examples/page
 
 go 1.24.7
 
 toolchain go1.24.11
 
 require (
-	github.com/jdetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
+	github.com/JDetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
 	github.com/pulumi/pulumi/sdk/v3 v3.212.0
 )
 

--- a/examples/quickstart/go/go.mod
+++ b/examples/quickstart/go/go.mod
@@ -1,11 +1,11 @@
-module github.com/jdetmar/pulumi-webflow/examples/quickstart/go
+module github.com/JDetmar/pulumi-webflow/examples/quickstart/go
 
 go 1.24.7
 
 toolchain go1.24.11
 
 require (
-	github.com/jdetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
+	github.com/JDetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
 	github.com/pulumi/pulumi/sdk/v3 v3.212.0
 )
 

--- a/examples/redirect/csharp/Webflow.Redirect.csproj
+++ b/examples/redirect/csharp/Webflow.Redirect.csproj
@@ -8,6 +8,6 @@
 
   <ItemGroup>
     <PackageReference Include="Pulumi" Version="3.0.0" />
-    <PackageReference Include="Pulumi.Webflow" Version="1.0.0-alpha.0" />
+    <PackageReference Include="Community.Pulumi.Webflow" Version="0.9.0" />
   </ItemGroup>
 </Project>

--- a/examples/redirect/go/go.mod
+++ b/examples/redirect/go/go.mod
@@ -1,11 +1,11 @@
-module github.com/jdetmar/pulumi-webflow/examples/redirect
+module github.com/JDetmar/pulumi-webflow/examples/redirect
 
 go 1.24.7
 
 toolchain go1.24.11
 
 require (
-	github.com/jdetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
+	github.com/JDetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
 	github.com/pulumi/pulumi/sdk/v3 v3.212.0
 )
 

--- a/examples/redirect/java/pom.xml
+++ b/examples/redirect/java/pom.xml
@@ -28,8 +28,8 @@
             <version>${pulumi.version}</version>
         </dependency>
         <dependency>
-            <groupId>com.pulumi</groupId>
-            <artifactId>webflow</artifactId>
+            <groupId>io.github.jdetmar</groupId>
+            <artifactId>pulumi-webflow</artifactId>
             <version>${webflow.version}</version>
         </dependency>
     </dependencies>

--- a/examples/robotstxt/csharp/Webflow.RobotsTxt.csproj
+++ b/examples/robotstxt/csharp/Webflow.RobotsTxt.csproj
@@ -8,6 +8,6 @@
 
   <ItemGroup>
     <PackageReference Include="Pulumi" Version="3.0.0" />
-    <PackageReference Include="Pulumi.Webflow" Version="1.0.0-alpha.0" />
+    <PackageReference Include="Community.Pulumi.Webflow" Version="0.9.0" />
   </ItemGroup>
 </Project>

--- a/examples/robotstxt/go/go.mod
+++ b/examples/robotstxt/go/go.mod
@@ -1,11 +1,11 @@
-module github.com/jdetmar/pulumi-webflow/examples/robotstxt
+module github.com/JDetmar/pulumi-webflow/examples/robotstxt
 
 go 1.24.7
 
 toolchain go1.24.11
 
 require (
-	github.com/jdetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
+	github.com/JDetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
 	github.com/pulumi/pulumi/sdk/v3 v3.212.0
 )
 

--- a/examples/robotstxt/java/pom.xml
+++ b/examples/robotstxt/java/pom.xml
@@ -28,8 +28,8 @@
             <version>${pulumi.version}</version>
         </dependency>
         <dependency>
-            <groupId>com.pulumi</groupId>
-            <artifactId>webflow</artifactId>
+            <groupId>io.github.jdetmar</groupId>
+            <artifactId>pulumi-webflow</artifactId>
             <version>${webflow.version}</version>
         </dependency>
     </dependencies>

--- a/examples/site/csharp/Webflow.Site.csproj
+++ b/examples/site/csharp/Webflow.Site.csproj
@@ -8,6 +8,6 @@
 
   <ItemGroup>
     <PackageReference Include="Pulumi" Version="3.0.0" />
-    <PackageReference Include="Pulumi.Webflow" Version="1.0.0-alpha.0" />
+    <PackageReference Include="Community.Pulumi.Webflow" Version="0.9.0" />
   </ItemGroup>
 </Project>

--- a/examples/site/go/go.mod
+++ b/examples/site/go/go.mod
@@ -1,11 +1,11 @@
-module github.com/jdetmar/pulumi-webflow/examples/site
+module github.com/JDetmar/pulumi-webflow/examples/site
 
 go 1.24.7
 
 toolchain go1.24.11
 
 require (
-	github.com/jdetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
+	github.com/JDetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
 	github.com/pulumi/pulumi/sdk/v3 v3.212.0
 )
 

--- a/examples/site/java/pom.xml
+++ b/examples/site/java/pom.xml
@@ -28,8 +28,8 @@
             <version>${pulumi.version}</version>
         </dependency>
         <dependency>
-            <groupId>com.pulumi</groupId>
-            <artifactId>webflow</artifactId>
+            <groupId>io.github.jdetmar</groupId>
+            <artifactId>pulumi-webflow</artifactId>
             <version>${webflow.version}</version>
         </dependency>
     </dependencies>

--- a/examples/stack-config/go-advanced/go.mod
+++ b/examples/stack-config/go-advanced/go.mod
@@ -1,11 +1,11 @@
-module github.com/jdetmar/pulumi-webflow/examples/stack-config/go-advanced
+module github.com/JDetmar/pulumi-webflow/examples/stack-config/go-advanced
 
 go 1.24.7
 
 toolchain go1.24.11
 
 require (
-	github.com/jdetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
+	github.com/JDetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
 	github.com/pulumi/pulumi/sdk/v3 v3.212.0
 )
 

--- a/examples/troubleshooting-logs/go-log-analysis/go.mod
+++ b/examples/troubleshooting-logs/go-log-analysis/go.mod
@@ -1,7 +1,7 @@
-module github.com/jdetmar/pulumi-webflow/examples/troubleshooting-logs/go-log-analysis
+module github.com/JDetmar/pulumi-webflow/examples/troubleshooting-logs/go-log-analysis
 
 go 1.21
 
 require github.com/pulumi/pulumi/sdk/v3 v3.100.0
 
-replace github.com/jdetmar/pulumi-webflow/sdk/go/webflow => ../../../sdk/go
+replace github.com/JDetmar/pulumi-webflow/sdk/go/webflow => ../../../sdk/go

--- a/examples/webhook/go/go.mod
+++ b/examples/webhook/go/go.mod
@@ -1,11 +1,11 @@
-module github.com/jdetmar/pulumi-webflow/examples/webhook
+module github.com/JDetmar/pulumi-webflow/examples/webhook
 
 go 1.24.7
 
 toolchain go1.24.11
 
 require (
-	github.com/jdetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
+	github.com/JDetmar/pulumi-webflow/sdk/go/webflow v0.0.1-alpha.1
 	github.com/pulumi/pulumi/sdk/v3 v3.212.0
 )
 

